cmd/kpod/create.go

```diff
@@ -343,6 +343,12 @@ func parseCreateOpts(c *cli.Context, runtime *libpod.Runtime) (*createConfig, er
 		blkioWeight = uint16(u)
 	}
 
+	if len(c.StringSlice("volume")) > 0 {
```

Comment:
No reason to check, just do the check in parseVolumes.

Reply:
fixed